packages/client/hmi-client/src/main.ts

```diff
@@ -44,11 +45,36 @@ app.component('math-field', MathfieldElement);
 app.component(VueFeather.name, VueFeather);
 app.use(VueKatex);
 
-const auth = useAuthStore();
-await auth.fetchSSO();
+const authStore = useAuthStore();
+const keycloak = new Keycloak('/api/keycloak/config');
+authStore.setKeycloak(keycloak);
 
-app.mount('body');
-logger.info('Application Mounted', { showToast: false, silent: true });
+await keycloak
+	.init({
+		onLoad: 'login-required' as KeycloakOnLoad
+	})
+	.then(async (auth) => {
```

Comment:
I don't see the point of using `await` and `then()`, either one or the other.

Reply:
fixed

Reply:
fixed